features/zoeken.feature

```diff
@@ -8,28 +8,89 @@ Functionaliteit: zoeken van WOZ-objecten
 
-  Scenario: zoek WOZ-object met adresseerbaar object identificatie van de aanduiding
-    Gegeven de LV WOZ kent een WOZ-object met objectnummer "051812345678"
-    En de aanduiding WOZ-object is ontleend aan adresseerbaar object identificatie "0518010000842214"
-    Als ik een WOZ-object zoek met /wozobjecten?adresseerbaarObjectIdentificatie=0518010000842214
-    Dan bevat het antwoord het WOZ-object met identificatie "051812345678"
-
-  Scenario: zoek WOZ-object met adresseerbaar object identificatie van een verbonden adresseerbaar object
-    Gegeven de LV WOZ kent een WOZ-object met objectnummer "051812345678"
-    En de aanduiding WOZ-object is ontleend aan adresseerbaar object identificatie "0518010000842214"
-    En in de LV WOZ is dit WOZ-object verbonden met adresseerbare objecten "0518010000842214", "0518010000609764" en "0518010000852970"
-    Als ik een WOZ-object zoek met /wozobjecten?adresseerbaarObjectIdentificatie=0518010000852970
-    Dan bevat het antwoord het WOZ-object met identificatie "051812345678"
-
-  Scenario: zoek WOZ-object met postcode en huisnummer van de aanduiding
-    Gegeven de LV WOZ kent een WOZ-object met objectnummer "051812345678"
-    En de aanduiding WOZ-object heeft postcode "2517GL" en huisnummer 84
-    Als ik een WOZ-object zoek met /wozobjecten?postcode=2517GL&huisnummer=84
-    Dan bevat het antwoord het WOZ-object met identificatie "051812345678"
-
-  Scenario: zoek WOZ-object met nummeraanduidingIdentificatie van de aanduiding
-    Gegeven de LV WOZ kent een WOZ-object met objectnummer "051812345678"
-    En de aanduiding WOZ-object heeft nummeraanduidingIdentificatie "0518200000842215"
-    Als ik een WOZ-object zoek met /wozobjecten?nummeraanduidingIdentificatie=0518200000842215
-    Dan bevat het antwoord het WOZ-object met identificatie "051812345678"
+  Regel: Zoeken met adresseerbaar object identificatie vindt WOZ-objecten waarbij de aanduiding WOZ-object is ontleend aan dit adresseerbaar object
+
+    Voorbeeld: zoek WOZ-object met adresseerbaar object identificatie van de aanduiding
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "051812345678"
+      En de aanduiding WOZ-object is ontleend aan adresseerbaar object identificatie "0518010000842214"
+      Als ik een WOZ-object zoek met /wozobjecten?adresseerbaarObjectIdentificatie=0518010000842214
+      Dan bevat het antwoord het WOZ-object met identificatie "051812345678"
+
+  Regel: Zoeken met adresseerbaar object identificatie vindt WOZ-objecten die verbonden zijn met dit adresseerbaar object
+
+    Voorbeeld: zoek WOZ-object met adresseerbaar object identificatie van een verbonden adresseerbaar object
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "051812345678"
+      En de aanduiding WOZ-object is ontleend aan adresseerbaar object identificatie "0518010000842214"
+      En in de LV WOZ is dit WOZ-object verbonden met adresseerbare objecten "0518010000842214", "0518010000609764" en "0518010000852970"
```

Comment:
Wat is het verschil tussen  `ontlenen van` en `verbonden met`? Als er verschil is tussen ontlenen en verbinden, kan een woz-object dan alleen met een verbonden adresseerbaar object id als één van de verbonden ao's het ao is waar het woz-object van is ontleend? Of is dat niet relevant? In dat geval kan de 'ontlenen van' stap worden verwijderd.

Reply:
zie discussie van overleg net. Volgens de beschrijving bij de parameter worden zowel objecten die zijn verbonden met de ado als objecten waarbij de aanduiding is ontleend aan gevonden.